tests/Http/Web/RockTheVoteReportTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Tests\Http\Web;
+
+use Tests\TestCase;
+use Illuminate\Support\Str;
+use Illuminate\Support\Facades\Bus;
+use Chompy\Jobs\ImportRockTheVoteReport;
+
+class RockTheVoteReportTest extends TestCase
+{
+    /**
+     * Test creating and importing a Rock The Vote Report via web.
+     *
+     * @return void
+     */
+    public function testCreateRockTheReportFormSubmission()
+    {
+        Bus::fake();
+
+        $this->rockTheVoteMock->shouldReceive('createReport')->andReturn((object) [
```

Comment:
Personally, I find it a little cleaner to use [`json_decode`](https://www.php.net/manual/en/function.json-decode.php)'s `$assoc` flag to return these as an associative array (and I think in general that's the pattern we follow elsewhere). That way you don't need to remember to cast this as an object elsewhere!

Reply:
Do you mean making this change over in  the `app/Services/RockTheVote:: createReport ` function, to return an array instead of an object?